### ISSUE TITLE
kiosk: MCP down in PACHA? 

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2665]
Still trying to deal with #2657 and PACHA's MCP is apparently down? Also didn't want to mess with it so I tell you.

<img width="1481" alt="Screenshot 2024-04-26 at 8 17 13 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/92dce214-6001-42d9-ae33-e8f1548913c2">

**Comment by urapadmin** [comment]
no, the messaging system of Kiosk is still confusing everybody (including me at times): It sends a message like an email if something turned out to be an issue at a certain point in time. As with an email, it might be obsolete by now, but it needs to be actively dismissed. This one is from 24. April 07:30, so a few days ago. 
I think the design of these alerts needs to change a bit in order to communicate the nature of the message. And maybe the wording. Hm. 

However, MCP is up. It can always be checked in administration under system information: 
![image](https://github.com/arch-kiosk/arch-kiosk-office/assets/38838314/8a39c742-b1f3-4fc6-8186-6040638671f1)

**Comment by luizaogs** [comment]
Ah woops ok, I will close that then. Do you want a new ticket about changing the alerts?

**Comment by luizaogs** [comment]
(I will open one, which you can close later if you don't want it.)
